### DISCUSSION TITLE
Editor does not add content to "p" tags, strips on save

### DISCUSSION THREAD

**Discussion by mauzilla** [tinymce/tinymce#9293] [General]
Using Umbraco 12 with TinyMCE 6 editor. Editor does not seem to recognize / insert "p" tags with no new line breaks. In the editor all seems correct, but upon saving, no "p" tag is surrounding the text and content is stored as 1 line. No errors in console either, any ideas?

**Comment by lostkeys** [comment]
Without a replication case its hard to troubleshoot. You can see if it is a TinyMCE issue by trying to provoke the issue in a [TinyMCE fiddle](https://fiddle.tiny.cloud). If you are unable to do it, it's probably better to ask your question where there are more Umbraco folks.

**Comment by mauzilla** [comment]
I managed to resolve it by removing the config and replaced it with config from a new installation, I suspect there was some config setting that could have caused it. Thanks for the reply.
